tests/integration/expected_test_results/api_v3_product_write/patch-request-fields-environmental_score-data.json

```diff
@@ -2,7 +2,7 @@
    "code" : "1234567890009",
    "errors" : [],
    "product" : {
-      "environmental_score_data" : {
+      "ecoscore_data" : {
```

Comment:
Is this related to images API or an accidental update of the test snapshot results?